tests/DbTestCase.php

```diff
@@ -362,4 +364,84 @@ protected function createTxtDocument(): Document
             ]
         );
     }
+
+    /**
+     * Helper method to enable a capacity on the given asset definition
+     *
+     * @param AssetDefinition $definition Asset definition
+     * @param string          $capacity   Capacity to enable
+     *
+     * @return AssetDefinition Updated asset definition
+     */
+    protected function enableCapacity(
+        AssetDefinition $definition,
+        string $capacity
+    ): AssetDefinition {
+        // Add new capacity
+        $capacities = json_decode($definition->fields['capacities']);
+        $capacities[] = $capacity;
+
+        $this->updateItem(
+            AssetDefinition::class,
+            $definition->getID(),
+            ['capacities' => $capacities],
+            // JSON encoded fields cannot be automatically checked
+            skip_fields: ['capacities']
+        );
+
+        // Reload asset definition
+        $definition->getFromDB(
+            $definition->getID()
+        );
+
+        // Force boostrap to trigger methods such as "onClassBootstrap"
+        $manager = AssetDefinitionManager::getInstance();
+        $this->callPrivateMethod(
+            $manager,
+            'boostrapConcreteClass',
+            $definition
+        );
+
+        return $definition;
+    }
+
+    /**
+     * Helper method to disable a capacity on the given asset definition
+     *
+     * @param AssetDefinition $definition Asset definition
+     * @param string          $capacity   Capacity to disable
+     *
+     * @return AssetDefinition Updated asset definition
+     */
+    protected function disableCapacity(
+        AssetDefinition $definition,
+        string $capacity
+    ): AssetDefinition {
+        // Remove capacity
+        $capacities = json_decode($definition->fields['capacities']);
+        $capacities = array_diff($capacities, [$capacity]);
+
+        $this->updateItem(
+            AssetDefinition::class,
+            $definition->getID(),
+            ['capacities' => $capacities],
+            // JSON encoded fields cannot be automatically checked
+            skip_fields: ['capacities']
+        );
```

Comment:
You should ensure that the capacity is correctly removed.
```suggestion
        );
        $this->array($this->callPrivateMethod($definition, 'getDecodedCapacitiesField'))->notContains($capacity);
```